### PR TITLE
Transition updated in FAQ

### PR DESCRIPTION
**Pull request by gauravchauhan2403**
@ankushdharkar  @swarajpure  Transitions have been updated in FAQ page. Please Review and merge. Suggest me the changes if needed.

### CODE REVIEW

File: css/pages/faq.css [hunk 1]
```diff
@@ -25,75 +20,50 @@ h1 {
...
-  color: #0000ff;
-  font-weight: 400;
-}
-
 .faq {
   background: #fff;
   border-radius: 0.5rem;
   box-shadow: 0 5px 15px rgba(0, 0, 0, 0.1);
-  padding: 1rem 1rem 0 1rem;
+  padding: 1rem 1rem 1rem 1rem;
```
Comment (swarajpure) [inline css/pages/faq.css:27]: Better to use just `padding: 1rem` 
`When one value is specified, it applies the same padding to all four sides.`
Source: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

**Review by swarajpure — CHANGES_REQUESTED** [review]
Great job here! Waiting for this one to be merged! 🤩

Suggesting a change regarding a line in CSS. Can be shortened.

File: css/pages/faq.css (hunk 1 above)
Comment (gauravchauhan2403) [reply css/pages/faq.css:27]: Ya sure @swarajpure . 
Thankyou for suggesting.

Comment (gauravchauhan2403) [reply css/pages/faq.css]: Change Done @swarajpure . May you Review it again.

Comment (ankushdharkar) [reply css/pages/faq.css]: Since all four values are the same, use `padding: 1rem;` @swarajpure @gauravchauhan2403

File: css/pages/faq.css [hunk 2]
```diff
@@ -117,21 +85,48 @@ a {
...
 }
 
-.slide-in-top {
-	-webkit-animation: slide-in-top 0.8s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
-          animation: slide-in-top 0.8s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
+footer {
+  position: relative;
+  height: 50px;
+  bottom: 0px;
+  left: 0px;
```
Comment (ankushdharkar) [inline css/pages/faq.css:127]: We should use one of them

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
Use either 'left' or 'right'. Both cannot be applied simultaneously

**Comment by gauravchauhan2403** [comment]
@ankushdharkar  All suggested changes have been done. Please Review and merge this.

**Comment by ankushdharkar** [comment]
Why use JS to achieve this effect?

**Review by ankushdharkar — APPROVED** [review]

**Pull request merged**